src/FieldWarning/Assets/Loading/CoroutineRunner.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CoroutineRunner : MonoBehaviour
```

Comment:
License + comment (something like "We need a monobehavior to start coroutines on so we use this empty one"), also no need to have the start and update methods.